src/Cli/dotnet/CommandFactory/CommandResolution/CompositeCommandResolver.cs

```diff
@@ -33,6 +36,12 @@ public CommandSpec Resolve(CommandResolverArguments commandResolverArguments)
 
                 if (commandSpec != null)
                 {
+                    TelemetryEventEntry.TrackEvent(CommandResolveEvent, new Dictionary<string, string>()
+                    {
+                        { "commandName", commandResolverArguments.CommandName.GetHashCode().ToString() },
```

Comment:
Don't hash like this - if you need to hash a value for telemetry purposes, you should use the `Sha256Hasher.HashWithNormalizedCasing` function to ensure consistency with all other telemetry value hashing.